src/components/GenericCard/genericCard.module.scss

```diff
@@ -0,0 +1,43 @@
+@import './../../constants/colors.scss';
+@import './../../constants/fonts.scss';
+
+.cardContainer {
+  background-color: $secondary-light-beige;
+  overflow: hidden;
+  width: fit-content;
+
+  .imageContainer {
+    width: 100%;
+    overflow: hidden;
+    height: fit-content;
+  }
+
+  .cardContent {
+    @media (min-width: 1440px) {
+      text-align: center;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+    }
+  }
+
+  .cardTitle {
+    font-family: $mobile-h3-font-family;
+    font-size: $mobile-h3-font-size;
+    font-weight: 400;
+    color: $mobile-h3-color;
+    letter-spacing: 2.67px; 
+    text-align: left;
+    margin: 0;
+    padding: 0;
+
+    @media (min-width: 1440px) {
+      font-family: $desktop-h3-font-family;
+      font-size: $desktop-h3-font-size;
+      font-weight: 400;
+      color: $desktop-h2-color;
```

Comment:
give the color more meaningfull names like - blue, blue100 ...
not h2 color because its not just color for h2